documentation.html

```diff
@@ -1393,14 +1396,69 @@ <h2 class="h2-style">Notification List</h2>
                     </p>
 
                 </div>
+                <hr>
+
+                <!-- Modal -->
+                <div id="modalaction">
+
+                    <h1 class="h1-style">Modal</h1>
```

Comment:
the class names could be improved here. Instead of `h1-styles`, you can use names such as `modal-header` or something similar. If in case  `h1-styles` is a utility class, kindly ignore the suggestion.

Reply:
its a utility class. Thanks.